### DISCUSSION TITLE
Twitter page offline?....but it's not.

### DISCUSSION THREAD

**Discussion by Linktydraa** [TumblThreeApp/TumblThree#424] [General]
Hi all,

I have been having this issue with Tumblthree on downloading some Twitter pages. I'll add the Twitter page to the queue, select download, and the app will look like it's about to download the page, but then it just doesn't. So hours will pass by, with the page sitting green and if it's downloading and then suddenly it'll error out saying the page is either offline or protected. So I'll go to the twitter page I'm not following but have in the blog, and the page will load just fine, not protected, offline, suspended, or anything. I have removed and readded the blog to the queue several times, and I keep getting the same error. I have zero ideas why Tumblthree would error saying a twitter page is offline or protected when it is clearly not. I also don't know why it'll sit "downloading" the blog for hours before erroring out. That is something that should only take a few minutes for the app to figure out.

**Comment by thomas694** [comment]
Hi,

for sure it shouldn't take hours to determine the status of a blog. But I never had a blog like that.
If you want you can post the URL of an affected blog in the feedback dialog and mention this topic number.
